dart/lib/src/protocol/sentry_span.dart

```diff
@@ -9,7 +9,7 @@ import '../utils.dart';
 
 class SentrySpan extends ISentrySpan {
   final SentrySpanContext _context;
-  DateTime? _timestamp;
+  DateTime? _endTimestamp;
```

Comment:
Can we keep `_timestamp` here and use `endTimestamp` only for the Public API? Since it's how the field is called anyway.